src/widgets/SymbolOverview.tsx

```diff
@@ -1,3 +1,142 @@
+    chartType: ChartType;
+    autosize: boolean;
+    symbols: ([string, string] | string)[]; // if in tuple form, use [alias, symbol]
+    backgroundColor: CSSProperties["backgroundColor"];
+    gridLineColor: CSSProperties["color"];
+    fontColor: CSSProperties["color"];
+    widgetFontColor: CSSProperties["color"];
+    hideLogo: boolean;
+    valuesTracking: "0" | "1" | "2" | "3";
+    volumeUpColor: CSSProperties["color"];
+    volumeDownColor: CSSProperties["color"];
+    noTimeScale: boolean;
+}
+
+export interface CandlesticksProps {
+    upColor: CSSProperties["color"];
+    downColor: CSSProperties["color"];
+    borderUpColor: CSSProperties["color"];
+    borderDownColor: CSSProperties["color"];
+    wickUpColor: CSSProperties["color"];
+    wickDownColor: CSSProperties["color"];
+}
+
+export interface BarsProps {
+    upColor: CSSProperties["color"];
+    downColor: CSSProperties["color"];
+}
+
+export interface LineProps {
+    color?: CSSProperties["color"];
+    colorGrowing?: CSSProperties["color"];
+    colorFalling?: CSSProperties["color"];
+    lineWidth?: number;
+}
+
+export interface AreaProps {
+    lineColor: CSSProperties["color"];
+    lineColorGrowing?: CSSProperties["color"];
+    lineColorFalling?: CSSProperties["color"];
+    topColor: CSSProperties["color"];
+    bottomColor: CSSProperties["color"];
+    lineWidth: number;
+}
+
+export type ChartType = "candlesticks" | "bars" | "line" | "area";
+
+export type MediumWidgetProps = MediumWidgetBaseOptions & (LineProps | AreaProps | BarsProps | CandlesticksProps) & {containerStyles?: CSSProperties}
+
+export enum ValuesTracking {
+	SCALE_LABELS = "0",
+	FLOATING_TOOLTIP = "1",
+	COLORED_TOOLTIP = "2",
+	LEGEND = "3",
+}
+
+const chartTypeDefaultValues: Record<ChartType, Record<string, string | number>> = {
+	bars: {
+		upColor: "#22AB94",
+		downColor: "#F7525F",
+	},
+	line: {
+		lineWidth: 3,
+	},
+	area: {
+		lineColor: "#2962FF",
+		topColor: "rgba(41, 98, 255, 0.3)",
+		bottomColor: "rgba(41, 98, 255, 0)",
+		lineWidth: 3,
+	},
+	candlesticks: {
+		upColor: "#22AB94",
+		downColor: "#F7525F",
+		borderUpColor: "#22AB94",
+		borderDownColor: "#F7525F",
+		wickUpColor: "#22AB94",
+		wickDownColor: "#F7525F",
+	}
+};
+
+export const SymbolOverview = React.forwardRef<HTMLDivElement, Partial<MediumWidgetProps & {fallback: React.ReactNode}>>(function SymbolOverviewRoot({ fallback, containerStyles, ...props }, ref) {
+	const idRef = useRef(getId());
+	const [ isError, setIsError ] = useState(false);
+
+	useEffect(() => {
+		(async () => {
+			try {
+				const TradingView = await injectTradingView();
+				new TradingView.MediumWidget({
+					chartOnly: false,
+					locale: "en",
+					colorTheme: "dark",
+					autosize: false,
+					showVolume: false,
+					hideDateRanges: false,
+					scalePosition:  "right",
+					scaleMode: "Normal",
+					fontFamily: "-apple-system, BlinkMacSystemFont, Trebuchet MS, Roboto, Ubuntu, sans-serif",
```

Comment:
i'm guessing there's a possibility someone could add their own `font-family` in the future (same thoughts with the other props) 🤔 something to consider as an improvement later on.

Reply:
If the user wants to add their own `font-family` prop, they can do so and it should overwrite the `"-applle-system..."` one since I've got the spread operator at the bottom!